dask/dataframe/tests/test_groupby.py

```diff
@@ -2189,6 +2189,38 @@ def test_groupby_transform_ufunc_partitioning(npartitions, indexed):
         )
 
 
+@pytest.mark.xfail(
+    reason=(
+        "Should work starting from pandas 1.0.0: "
+        "https://github.com/dask/dask/pull/5423"
+    ),
+    strict=False,
```

Comment:
We want `strict=True` here, otherwise no one will notice that it's actually working now.

I think the skip will need to be conditional on the pandas version. The skip should only apply to `not dask.datafraue.utils.PANDAS_GT_100`

Reply:
Thanks for your feedback @TomAugspurger! I had to change around the parametrization a bit because only 1 of the input sets fails at the moment.